analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -231,6 +231,19 @@ class Cargo(
         return null
     }
 
+    /**
+     * Extract information about authors from the given [node] with package metadata.
+     */
+    private fun parseAuthors(node: JsonNode?): SortedSet<String> =
+        node?.mapNotNullTo(sortedSetOf()) { authorNode -> parseAuthorString(authorNode.textValue()) } ?: sortedSetOf()
```

Comment:
Nit: For such one-liners we usually stick to using the default `it` in the lambda.